src/vm/methodtablebuilder.cpp

```diff
@@ -5961,6 +5961,12 @@ MethodTableBuilder::InitMethodDesc(
             pNewNMD->ndirect.m_cbStackArgumentSize = 0xFFFF;
 #endif // defined(_TARGET_X86_)
 
+            if (RVA != 0 && IsMiUnmanaged(dwImplFlags) && IsMiNative(dwImplFlags))
```

Comment:
This looks like a specific check for IJW - i.e. RVA is non-zero and it is unmanaged and native. Where as a P/Invoke is basically RVA is zero and unmanaged and native?

If that is true, can we add a comment this is a IJW check?

Reply:
Done.